engine/admin/logging/awslogs.md

```diff
@@ -77,6 +77,115 @@ $ docker run --log-driver=awslogs \
 > **Note**:
 > Your AWS IAM policy must include the `logs:CreateLogGroup` permission before you attempt to use `awslogs-create-group`.
 
+### awslogs-multiline-pattern
+
+The `awslogs-multiline-pattern` specifies a regular expression pattern for identifying the start of a log message.  A log message is made of a line that matches the pattern and any following lines that don't match the pattern.  
```

Comment:
The `awslogs-multiline-pattern` specifies a regular expression pattern for identifying the beginning of a container log entry.  Any following lines that do not match the pattern are considered to be part of the current log entry. The next line that matches the pattern begins a new log entry.